src/renderers/shared/stack/reconciler/__tests__/ReactMultiChild-test.js

```diff
@@ -148,5 +148,60 @@ describe('ReactMultiChild', function() {
       expect(mockMount.mock.calls.length).toBe(2);
       expect(mockUnmount.mock.calls.length).toBe(1);
     });
+
+    it('should warn for duplicated keys with component stack info', function() {
+      // NOTE: We're explicitly not using JSX here. This is intended to test
```

Comment:
Again, let's use JSX and do a regex replace to normalize it so we can check equality.